pysmt/test/test_euf.py

```diff
@@ -37,6 +37,22 @@ def test_euf(self):
         self.assertSat(check, logic=UFLIRA,
                        msg="Formula was expected to be sat")
 
+    @skipIfNoSolverForLogic(UFLIRA)
+    def test_infix(self):
+        ftype1 = FunctionType(REAL, [REAL])
+        ftype2 = FunctionType(REAL, [REAL, INT])
```

Comment:
Can you add a test case for f: Int x BV8 -> Real? It seems to me that the implementation that you provide should not work with BV. When you convert the args, you should look into the function type definition and for each positional argument detect the type... I am confused on why this works for INT...

Reply:
Done.